apps/asap-server/src/handlers/webhooks/fetch-by-code/fetch-by-code.ts

```diff
@@ -31,16 +30,9 @@ export const fetchUserByCodeHandlerFactory = (
     };
 
     const user = await userController.fetchByCode(code);
-    let apiKey: string | null;
-
-    try {
-      apiKey = algoliaClient.generateSecuredApiKey(algoliaSearchApiKey, {
-        validUntil: Date.now() + 36060, // which is one minute over the TTL of the ID token
-      });
-    } catch (e) {
-      apiKey = null;
-      logger.error(e, 'Error during generating the key for Algolia search');
-    }
+    const apiKey = algoliaClient.generateSecuredApiKey(algoliaSearchApiKey, {
+      validUntil: Date.now() + 36060, // which is one minute over the TTL of the ID token
```

Comment:
Think there was another comment about this - can we put the `36060` into a `const`?

Reply:
we could but erm.. what difference does it make :D? i guess i can put it in the config although to be fair its not a config attribute you would ever need to change and if you do its still global. But yeah if thats what you guys want